bioindex/src/main/resources/variants.py

```diff
@@ -0,0 +1,31 @@
+from pyspark.sql import SparkSession, Row
+from pyspark.sql.functions import col, concat_ws, explode, regexp_replace, row_number, split
+from pyspark.sql.types import IntegerType
+
+
+def main():
+    """
+    Arguments: none
+    """
+    spark = SparkSession.builder.appName('bioindex').getOrCreate()
+
+    srcdir = 's3://dig-analysis-data/out/varianteffect/common/part-*'
+    outdir = 's3://dig-bio-index/variants/common/'
+
+    df = spark.read.json(srcdir)
+
+    # add chromosome and position
+    df = df.withColumn('chromosome',  split(df.varId, ':').getItem(0)) \
```

Comment:
I'm assuming here that we'll just leave the dbSNP field intact even though we'll no longer support queries against it directly.